translations/monero-core_pl.ts

```diff
@@ -26,42 +26,42 @@
     <message>
         <location filename="../pages/AddressBook.qml" line="81"/>
         <source>&lt;b&gt;Payment ID&lt;/b&gt;&lt;br/&gt;&lt;br/&gt;A unique user name used in&lt;br/&gt;the address book. It is not a&lt;br/&gt;transfer of information sent&lt;br/&gt;during the transfer</source>
-        <translation type="unfinished"></translation>
+        <translation>&lt;b&gt;Identyfikator płatności&lt;/b&gt;&lt;br/&gt;&lt;br/&gt;Unikalna nazwa użytkownika użyta w&lt;br/&gt;książce adresowej. Nie jest to transfer informacji&lt;br/&gt;wysłanej podczas transferu</translation>
     </message>
     <message>
         <location filename="../pages/AddressBook.qml" line="88"/>
         <source>Paste 64 hexadecimal characters</source>
-        <translation type="unfinished">Wklej 64 znaki w postaci heksadecymalnej (dodaj na początku adresu &apos;0x&apos;))</translation>
```

Comment:
Yeah. Good that someone marked it as "unfinished" because frankly I didn't check fully translated phrases. I plan to make a PR after the hard fork that will modify a bit English phrases (as szogun1987) made some good suggestions and then check all Polish phrases and make another PR.